modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectValidationRuleResourceImpl.java

```diff
@@ -68,6 +71,10 @@ public void deleteObjectValidationRule(Long objectValidationRuleId)
 			objectDefinition.getObjectDefinitionId(), search, pagination);
 	}
 
+	@NestedField(
+		parentClass = com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition.class,
```

Comment:
I think you could replace the path to `ObjectDefinition.class`. It is a minor change, so I don't see it as a required.

Reply:
Hey @marcelabc. In this case, it's because we already have ObjectDefinition imported from object-api. That's why we needed to put the full path of it. Thx for the review.